src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -297,12 +301,11 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return this.quote.quote.cryptoAmount
   }
 
-  getCryptoTypeString(): string {
-    // todo eventually rename this to getCryptoType and rename getCryptoType to getCryptoCurrency
+  getCryptoType(): string {
```

Comment:
I think that previous comment was a bit misleading; this should technically return a `CryptoType` enum from `fiatconnect-types`, in alignment with what we do with `getFiatType`.

Reply:
although i wonder if TS will complain since the interface calls for `getCryptoType(): string`

Reply:
TS was okay with it in the file, but complained in other files where it was accessed and compared and expected to be a string.